ui-lib/src/main/java/cash/z/ecc/ui/screen/home/model/WalletSnapshot.kt

```diff
@@ -27,3 +27,9 @@ fun WalletSnapshot.totalBalance(): Zatoshi {
 
     return Zatoshi(total.coerceAtLeast(0))
 }
+
+fun WalletSnapshot.spendableBalance(): Zatoshi {
+    val total = (orchardBalance + saplingBalance).totalZatoshi
```

Comment:
NOTE: This is yet to be determined. Spending from both pools has side-effects that users need to be aware of. 
Funds jumping pools, reveal the value being transferred from one pool to another. Although (as I understand and might be wrong about it) the protocol allows a transaction that combines funds from different pools, I'm not sure if we want that to happen in our SDKs. Probably I'd say that it's for the best interest of the users' privacy that funds are migrated to the best-pool (orchard in this case) before they can be spendable.

Reply:
I can add an inline comment in the code to clarify that this is provisional, that way someone looking at it later will know.